sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1205,14 +1205,13 @@ void _pi_queue::adjustBatchSizeForFullBatch(bool IsCopy) {
     CommandBatch.NumTimesClosedEarly = 0;
     CommandBatch.NumTimesClosedFull = 0;
   }
-  CommandBatch.QueueBatchSize = QueueBatchSize;
 }
 
 void _pi_queue::adjustBatchSizeForPartialBatch(bool IsCopy) {
   auto &CommandBatch = IsCopy ? CopyCommandBatch : ComputeCommandBatch;
   auto &ZeCommandListBatchConfig =
       IsCopy ? ZeCommandListBatchCopyConfig : ZeCommandListBatchComputeConfig;
-  pi_uint32 QueueBatchSize = CommandBatch.QueueBatchSize;
+  pi_uint32 &QueueBatchSize = CommandBatch.QueueBatchSize;
```

Comment:
ouch, does it mean the dynamic batching wasn't really working? pretty sure this will get perf swings, let's try to see them in pre-commit